app/Services/Forms/QuestionsService.php

```diff
@@ -72,9 +72,20 @@ public function updateQuestion(int $question_id, array $question)
         // そのため、CodeIgniter のバリデーションの邪魔になる文字列は
         // 削除する
         // TODO: CodeIgniter を廃止したら、以下の処理は削除する
```

Comment:
```php
// CodeIgniter 側の in_list バリデーションルールを使いたいが、
// CodeIgniter ではバリデーションルールを単なる文字列で表す。
// そのため、CodeIgniter のバリデーションの邪魔になる文字列は
// 削除する
// TODO: CodeIgniter を廃止したら、以下の処理は削除する
```

コメントの位置がここだと、CodeIgniter が廃止された際、重複した選択肢を削除する処理も一緒に消してしまいそうで怖いので、コメントを書き換えて欲しいです

```suggestion
        // TODO: CodeIgniter を廃止したら、str_replace をする処理は削除する
```

とかかな